### ISSUE TITLE
Default some values rather than require constants or other values

### ISSUE THREAD

**Issue by JBerendes** [shahen94/react-native-video-processing#77]
This is certainly an enhancement request and I would be willing to help with the updates but would require a brief discussion.

To keep the code little more lean consider the best/common use case and set default values.
example:
Dont't make the dev write::
```
<VideoPlayer
ref={ref => this.videoPlayerRef = ref}
startTime={30}  // seconds
endTime={120}   // seconds
play={true}     // default false
replay={true}   // should player play video again if it's ended
rotate={true}   // use this prop to rotate video if it captured in landscape mode iOS only
source={'file:///sdcard/DCIM/....'}
playerWidth={300} // iOS only
playerHeight={500} // iOS only
style={{ backgroundColor: 'black' }}
resizeMode={VideoPlayer.Constants.resizeMode.CONTAIN}
onChange={({ nativeEvent }) => console.log({ nativeEvent })} // get Current time on every second
/>
```
when defaults would allow them to write::
```
<VideoPlayer
ref={ref => this.videoPlayerRef = ref}
source={'file:///sdcard/DCIM/....'}
onChange={({ nativeEvent }) => console.log({ nativeEvent })} // get Current time on every second
/>
```
Defaults: startTime=0, endTime=null, play=true, replay=false, rotate=true, playerWidth=window.width, payerHeight=window.height, style=null, resizeMode=VideoPlayer.Constants.resizeMode.COVER
(these defaults represent my best guess)

Everything here is just a recommendation and I hope you will consider it.

**Comment by shahen94** [comment]
I'm totally fine with this suggestions, but for some users this can be breaking change right.
If you want to open PR for this - i'll recommend you to commit it with prefix `BREAKING CHANGE`

**Comment by JBerendes** [comment]
I wouldn't see this as a breaking change unless I'm misrepresenting some values that already have defaults set. Leave any current defaults as is. Does this seem correct?
This really comes to light for me because I started getting a warning when the resizeMode constant was introduced. Having a warning come that I had to dismiss and then deal with was what I considered a breaking change. Am I making sense? :)
